smart-contracts/wasm-chain-integration/src/v1/ffi.rs

```diff
@@ -103,8 +104,9 @@ unsafe extern "C" fn call_init_v1(
     // is initialized. However reflecting this in types would be a lot of extra work for no
     // real gain. So we require it.
     loader: LoadCallback,
```

Comment:
The doc comment of this function still talks about artifact pointers, which no longer exist. Please update.